### DISCUSSION TITLE
Specify name of sender ('Display name') in e-mail client

### DISCUSSION THREAD

**Discussion by notDavid** [simple-login/app#1064] [Q&A] [answered]
Hi there,

So if i create an alias and leave the `Display name` field empty, is it then possible to somehow specify the Display name when sending the emails from Thunderbird, by setting it in the `From` field or something?

Please advise,
Thank you!

**Comment by nguyenkims** [comment, marked as answer]
@notDavid No as SimpleLogin will create a completely new From: using the alias and its display name.

**Reply by notDavid** [reply]
Okay I see... Thank you :-)

**Reply by benjuade** [reply]
@notDavid the original data are saved as email headers. I found this plugin that may help you (and me) in having a quick info on the sender email (as happens in the proton UI web interface): https://github.com/oktupol/additional-info-for-simplelogin-thunderbird

If you use Proton, also this one is useful: https://github.com/Dan-Q/ProtonMail-Encryption-Indicators-for-Thunderbird
